packages/vuetify/src/components/VList/VList.tsx

```diff
@@ -108,6 +108,12 @@ export const VList = genericComponent<new <T>() => {
     ...makeThemeProps(),
   },
 
+  emits: {
+    'update:selected': (val: string[]) => true,
+    'update:opened': (val: string[]) => true,
+    'update:active': (val: string[]) => true,
```

Comment:
```suggestion
    'update:active': (val: string[]) => true,
    'update:opened': (val: string[]) => true,
    'update:selected': (val: string[]) => true,
```

Reply:
This is the same order as in the original interface.